.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Commitlint
+on: [push, pull_request]  # yamllint disable-line rule:truthy
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
this only checks out the last commit, you'll need a more complete history to check each commit